### ISSUE TITLE
BusyIndicator focus throws null reference Error

### ISSUE THREAD

**Issue by codefactor** [SAP/ui5-webcomponents#4504]
### **Bug Description**
Opening a Dialog with a BusyIndicator that is not yet met the 1 second threshold will cause a null reference error when opening the dialog. 

### **Expected Behavior**
The error should not occur.

### **Steps to Reproduce**
Create a dialog element with a BusyIndicator that is created "on the fly" and then open the dialog immediately. There is a null reference error that occurs. Using the isolated example below, simply press the "Open Dialog" button to reproduce the problem.

### **Isolated Example**
https://codesandbox.io/s/ui5-webcomponents-forked-0i79j?file=/src/index.js

### **Context**
- UI5 Web Components version: 1.0.2
- Affected component: BusyIndicator

### **Log Output / Stack Trace / Screenshots**
```
    at HTMLElement._redirectFocus (https://0i79j.csb.app/node_modules/
ui5/webcomponents/dist/BusyIndicator.js:144:67
EventPart.handleEvent
https://0i79j.csb.app/node_modules/lit-html/development/lit-html.js:674:29
    at HTMLElement.applyFocus (https://0i79j.csb.app/node_modules/
ui5/webcomponents/dist/Popup.js:190:15
    at async HTMLElement.applyInitialFocus (https://0i79j.csb.app/node_modules/
ui5/webcomponents/dist/Popup.js:181:5
```
### **Priority**
- [ ] Low
- [ ] Medium
- [x] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactors
- Bussiness impact: This causes the entire React app to disappear.

**Comment by codefactor** [comment]
This is a common use case to show a dialog with a busy indicator.

**Comment by vladitasev** [comment]
Turns out this was fixed already by: https://github.com/SAP/ui5-webcomponents/pull/4321

Regards,
Vladi

**Comment by hkshetry** [comment]
@vladitasev  can you please confirm the release version?

**Comment by fifoosid** [comment]
Hi @hkshetry 

This fix will be released with 1.1.0 version
